com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfCollisionExtensions.cs

```diff
@@ -42,7 +41,9 @@ public static void Create(Transform parent, GeometryTypes type, Transform visual
             else
                 UrdfGeometryCollision.Create(collisionObject.transform, type);
 
+#if UNITY_EDITOR
             UnityEditor.EditorGUIUtility.PingObject(collisionObject);
```

Comment:
Just curious why do we need to ping the object in the scene?

Reply:
This is to highlight the newly created game object in the inspector: